src/hotspot/os/posix/signals_posix.cpp

```diff
@@ -1372,7 +1349,7 @@ void PosixSignals::print_signal_handler(outputStream* st, int sig,
   // See comment for SA_RESTORER_FLAG_MASK
   LINUX_ONLY(sa.sa_flags &= SA_RESTORER_FLAG_MASK;)
 
-  st->print("%s: ", os::exception_name(sig, buf, buflen));
+  st->print("%10s: ", os::exception_name(sig, buf, buflen));
```

Comment:
Where does the 10 come from? Why do we need to limit it?

Reply:
> A minor comment, but otherwise this seems quite reasonable.
> 
> Thanks,
> David

Thanks, David!

`%10s` is not a limit, its for alignment. It makes all the signal names nicely aligned (all colons end up in the same column).

..Thomas